buildSrc/DesktopBuilder.js

```diff
@@ -3,15 +3,14 @@ const babel = Promise.promisifyAll(require("babel-core"))
 const fs = Promise.promisifyAll(require("fs-extra"))
 const path = require("path")
 
-
-function build(dirname, version, targets, updateUrl, nameSuffix, notarize) {
+function build(dirname, version, targets, updateUrl, nameSuffix, notarize, outDir, unpacked) {
```

Comment:
I think it would be best to make it options object, it's absolutely impossible to keep all the arguments in mind when you see the invocation

Reply:
That was my feeling as well, I just wasn't sure if that's worth it when the Rollup-migration changes everything again. Since that's still indefinitely postponed, I guess it would improve it enough.

Reply:
I'm not sure this part will change, we will anyway need to transmit all those arguments to the desktop build, Rollup or not